### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by frontdevde**
minor: fixed some typos in the project's readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ PostCSS transforms are often more powerful than
 
 ## Usage
 
-You need to make 2 steps for using PostCSS:
+You just need to follow these two steps to be able to use PostCSS:
```
Comment (ben-eb) [inline README.md]: *to use* ?

File: README.md [hunk 2]
```diff
@@ -125,7 +125,7 @@ gulp.task('css', function () {
 });
 ```
 
-For other environments your can use [CLI tool] or JS API:
+For other environments your can use the [CLI tool] or the JS API:
```
Comment (ben-eb) [inline README.md]: Can you fix the *your* in this sentence to *you*?

**Comment by ben-eb** [comment]
Other than the comments, :+1:

**Comment by frontdevde** [comment]
@ben-eb good shout, amended it

**Comment by ben-eb** [comment]
Thanks for your contribution. :smile:

**Pull request merged**